projects/vdk-core/src/vdk/internal/cli_entry.py

```diff
@@ -128,12 +128,15 @@ def vdk_main(
         exit_code = 0
         try:
             log.info(f"Start CLI {program_name} with args {command_line_args}")
-            exit_code = cast(InternalHookSpecs, plugin_registry.hook()).vdk_cli_execute(
-                root_command=root_command,
-                command_line_args=command_line_args,
-                core_context=core_context,
-                program_name=program_name,
-            )
+            if root_command is not None:
+                exit_code = cast(
+                    InternalHookSpecs, plugin_registry.hook()
+                ).vdk_cli_execute(
+                    root_command=root_command,
+                    command_line_args=command_line_args,
+                    core_context=core_context,
+                    program_name=program_name,
+                )
             return exit_code
```

Comment:
In case of failure, `vdk` returns != 0 by design. That's a major change, modifying the application exit status.

Reply:
I saw the change; maybe initialise with 1, to save workflow branches (else)